takwimu/models/dashboard.py

```diff
@@ -599,6 +682,106 @@ def __str__(self):
         return self.question.encode('ascii', 'ignore')
 
 
+class FeaturedDataBlock(blocks.StructBlock):
+    title = blocks.CharBlock(required=False)
+    country = blocks.ChoiceBlock(required=True,
+                                 choices=[
+                                     ('ET', 'Ethiopia'),
+                                     ('KE', 'Kenya'),
+                                     ('NG', 'Nigeria'),
+                                     ('SN', 'Senegal'),
+                                     ('TZ', 'Tanzania'),
+                                 ],
+                                 label='Country')
+    data_id = blocks.ChoiceBlock(required=True,
+                                 choices=HURUMAP_DATA_DISTS,
+                                 label='Data')
+    chart_type = blocks.ChoiceBlock(required=True,
+                                    choices=[
+                                        ('histogram', 'Histogram'),
+                                        ('pie', 'Pie Chart'),
+                                        ('grouped_column',
+                                         'Grouped Column')
+                                    ],
+                                    label='Chart Type')
+    data_stat_type = blocks.ChoiceBlock(required=True,
+                                        choices=[
+                                            ('percentage',
+                                             'Percentage'),
+                                            ('scaled-percentage',
+                                             'Scaled Percentage'),
+                                            ('dollar', 'Dollar')
+                                        ],
+                                        label='Stat Type')
+    chart_height = blocks.IntegerBlock(required=False,
+                                       label='Chart Height',
+                                       help_text='Default is 300px')
+    description = blocks.TextBlock(
+        required=False, label='Description of the data')
+
+
+class PageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ProfileSectionPage
+        fields = ['title', 'description']
+
+
+class FeaturedAnalysisBlock(blocks.StructBlock):
+    name = 'featured_analysis'
+    from_country = blocks.ChoiceBlock(choices=country_choices)
+    featured_page = blocks.PageChooserBlock(target_model=ProfileSectionPage)
+
+    def get_api_representation(self, value, context=None):
```

Comment:
@karimkawambwa just doing last minutes checks before merging: Why didn't we enable custom serializer via `api_fields = [ ... , serializer=.... ]` i.e the Wagtail `v2` way: https://docs.wagtail.io/en/v2.3/advanced_topics/api/v2/configuration.html#custom-serialisers

Reply:
Interesting, let me give it a try.
This way I guess FeaturedAnalysisBlock stays with default serialization if used anywhere else.

Reply:
Cool. Lets do that on separate PR @karimkawambwa.